src/index.js

```diff
@@ -7,6 +7,8 @@ const pump = require('pump')
 
 const HUB_URL = 'localhost:8080'
 
+const dbExists = require('./indexedDBUtils').dbExists
```

Comment:
Please move just before HUB_URL

Reply:
Follow the rule:
At the very top: node_modules imports
one line after: relative imports